src/models/TranslationValue/addTranslationValueToTranslationKey/index.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+import TranslationValue from '@models/TranslationValue'
+import TranslationKey from '@models/TranslationKey'
+import Locale from '@models/Locale'
+import validateString from '@helpers/validateString'
+import type { ValidationError } from '@types/ValidationError'
+import { DUPLICATE_ENTRY_ERROR_TYPE } from '@constants/errors'
+
+const validateValue = validateString({
+  type: 'translation value',
+  minLenght: 1,
+  maxLength: Infinity
+})
+
+export type AddTranslationValueToTranslationKeyArgs = {
+  value: string,
+  translationKeyId: string,
+  localeId: string,
+  ownerId: string
+}
+
+type AddTranslationValueToTranslationKeyResponse = {
+  translationKey?: TranslationKey,
+  errors: Array<ValidationError>
+}
+
+const addTranslationValueToTranslationKey = async ({
+  value,
+  translationKeyId,
+  localeId,
+  ownerId
+}: AddTranslationValueToTranslationKeyArgs): Promise<
+  AddTranslationValueToTranslationKeyResponse
+> => {
+  const errors: Array<ValidationError> = []
+
+  const translationKey: TranslationKey = await TranslationKey.query()
+    .findById(translationKeyId)
+    .eager('[project.locales, translationValues.locale]')
+
+  if (!translationKey) {
+    throw new Error('This translation key was not found.')
+  }
+
+  const project = translationKey.project
+  const locale = await Locale.query().findById(localeId)
+
+  if (project && !project.hasOwnerId(ownerId)) {
+    throw new Error('This project was not found.')
+  }
+
+  if (!locale) {
+    throw new Error('This locale doesn’t exist.')
+  }
+
+  let isLocaleActivated = false
+  project.locales.forEach(projectLocale => {
+    if (projectLocale.id === locale.id) {
+      isLocaleActivated = true
+    }
+  })
+
+  if (!isLocaleActivated) {
+    throw new Error('This locale is not activated for this project')
+  }
+
+  const valueValidation = validateValue(value)
+
+  if (!valueValidation.isValid) {
+    errors.push({
+      field: 'value',
```

Comment:
`translation value`, no?

Reply:
The field on the mutation is `value`, so, no ? 😅  that's the field not the content of the error.
validateValue is : 
```
const validateValue = validateString({
  type: 'translation value',
  minLength: 1,
  maxLength: Infinity
})
```

Reply:
oh yeah sorry mixed the `type` with the `field`, nevermind 😄